master/static/srv/update_aggregated_allocation.sql

```diff
@@ -5,67 +5,45 @@ WITH const AS (
             ($1 :: timestamptz + interval '1 day')
         ) AS period
 ),
--- Workloads that had any overlap with the target interval, along with the length of the overlap of
+-- Allocations that had any overlap with the target interval, along with the length of the overlap of
 -- their time with the requested period.
-workloads AS (
+allocs_in_range AS (
     SELECT
-        all_workloads.trial_id,
+        a.*,
         extract(
             epoch
             FROM
                 -- `*` computes the intersection of the two ranges.
-                upper(const.period * range) - lower(const.period * range)
-        ) * (
-            experiments.config -> 'resources' ->> 'slots_per_trial'
-        ) :: float AS seconds
+                upper(const.period * a.range) - lower(const.period * a.range)
+        ) * (a.slots) :: float AS seconds
     FROM
         (
             SELECT
-                trial_id,
+                *,
```

Comment:
I'm really in favor of * here, since semantically this is "select all allocations with the range precalculated".. it's essentially just a view of the allocations table. if we didn't use *, this would need to be changed any time the table changed.